gpt4all-backend/replit.cpp

```diff
@@ -144,12 +144,13 @@ std::vector<LLModel::Token> replit_tokenizer_tokenize(replit_tokenizer & tokeniz
     return tokenized.first;
 }
 
-std::string replit_tokenizer_detokenize(replit_tokenizer & tokenizer, const std::vector<LLModel::Token> & tokens) {
+std::string_view replit_tokenizer_detokenize(replit_tokenizer & tokenizer, const std::vector<LLModel::Token> & tokens) {
```

Comment:
This use of stringview was problematic and leading to leaks

Reply:
Argh, mistake during rebase